sentry.gradle

```
@@ -165,7 +165,8 @@ def extractReleasesInfo() {
 
         variant.outputs.each { output ->
             def versionCode = output.getVersionCode()
```

```suggestion
            def defaultVersionCode = output.getVersionCode()
            def versionCode = System.getenv("SENTRY_DIST") ?: defaultVersionCode
```
We would need the `dist` to be configurable as well.

---

done